README.md

```diff
@@ -84,14 +84,22 @@ The following instructions will help you get started with a working SaaS wallet
 
 ### Prerequisites
 1. An Azure subscription
-2. Create a new Azure Active Directory application. This application will be used for authenticating against Azure. 
-[Setting AAD application](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-integrating-applications)
-3. Get the service principal id and secret for the application from step 2
+2. Setup an Azure Active Directory application
+3. Prepare the [Service principal id and secret and assign it to a contributor role](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal)
+4. An Azure Active Directory TenantId - Same link as step 3
+5. Prepare the Azure Service principal object id
+   1. Go to the azure portal
+   2. go to Azure Active Directory -> Enterprise Applications -> All applications. 
```

Comment:
nitpick: all sentence should start with capital :)

Reply:
👍